ci/build.ps1

```diff
@@ -36,20 +35,20 @@ $scoop = (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh'
   Invoke-Expression $scoop
 }
 
-scoop install perl
-perl --version
-cpanm.bat --version
+# scoop install perl@5.30.3.1
```

Comment:
What's the error?

Reply:
Something about overwriting `Config::STORE`.
But I'll try to reimolement the script as a GH workflow instead, to see how it goes.